packages/dashboard/src/components/datagrid/useColumnOrder.tsx

```
@@ -0,0 +1,39 @@
+      // fall through
+    }
+    return defaultKeys;
+  });
+
+  const reorderColumns = useCallback(
+    (sourceKey: string, targetKey: string) => {
+      setColumnKeys((prev: string[]) => {
+        const next = [...prev];
+        const from = next.indexOf(sourceKey);
+        const to = next.indexOf(targetKey);
+        if (from === -1 || to === -1) return prev;
```

ESLint

[error] Expected { after 'if' condition.

(curly)